sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -6934,3 +6936,29 @@ def test_is_combining():
 def test_issue_17857():
     assert pretty(Range(-oo, oo)) == '{..., -1, 0, 1, ...}'
     assert pretty(Range(oo, -oo, -1)) == '{..., 1, 0, -1, ...}'
+
+def test_issue_18272():
+    x = Symbol('x')
+    n = Symbol('n')
+
+    assert upretty(solveset(exp(x) - x, x)) == \
+    '⎧            ⎛      x    ⎞⎫\n'\
+    '⎨x | x ∊ ℂ ∧ ⎝-x + ℯ  = 0⎠⎬\n'\
+    '⎩                         ⎭'
+    assert upretty(solveset(Abs(2*x) - n, x, S.Reals)) == \
+    '⎧        ⎧-n   n⎫   ⎛n         ⎞⎫\n'\
+    '⎨x | x ∊ ⎨───, ─⎬ ∧ ⎜─ ∈ [0, ∞)⎟⎬\n'\
+    '⎩        ⎩ 2   2⎭   ⎝2         ⎠⎭'
```

Comment:
Can you make it so that this doesn't call solveset but creates the set directly?

E.g.:
```julia
In [1]: solveset(Abs(2*x) - n, x, S.Reals)                                                                                                     
Out[1]: 
⎧        ⎧-n   n⎫                                       ⎫
⎨x | x ∊ ⎨───, ─⎬ ∧ (n         , ─ ∈ [0, ∞), 2         )⎬
⎩        ⎩ 2   2⎭                                       ⎭

In [2]: repr(_)                                                                                                                                
Out[2]: 'ConditionSet(x, Contains(n/2, Interval(0, oo)), FiniteSet(-n/2, n/2))'
```
You can use the repr output as input for this test.

Reply:
Ok, I'll update it. Also, Can you please give any suggestions to increase the diff coverage for the report?

Reply:
> Can you please give any suggestions to increase the diff coverage for the report?

The codecov report I see says that the diff coverage is 100%. I think that's probably good enough :)